racksdb/web/ui/src/components/InfrastructurePopUp.vue

```diff
@@ -1,98 +1,41 @@
-<!--Copyright (c) 2022-2023 Rackslab
```

Comment:
You introduce the notion of _PopUp_ for a component generally named _modal_ (or _dialog_) by the community, can you please rename it to use the same name as the community?

Reply:
I replaced all popup mentions by modal.